api/src/main/resources/workbench-api.yaml

```diff
@@ -5345,6 +5385,18 @@ definitions:
         format: int64
         description: 'Timestamp when the user first changed from Unregistered to Registered
           data access level. May be null if the user has never done so.'
+      rasLinkLoginGovName:
+        type: string
+        description: The user's identity name in login.gov.
+      rasLinkLoginGovExpireTime:
+        type: integer
+        format: int64
+        description: The user's login.gov/AoU link expiration time
+      rasLinkLoginGovCompletionTime:
```

Comment:
Does this *only* show up if the user has linked an IAL2-enabled account? Or can it show up otherwise? I would probably include this in the property name (maybe `rasLinkLoginGovIal2CompletionTime`), or at least mention this in the comment - since it seems like the access requirement we'll want to track is based on "the time at which we confirmed a linkage to an IAL2 login.gov account"

Reply:
right, our target is Login.gov + IAL2 at the same, and there will be no such the case in our DB, profile that user linkede login.gov without IAL2. (we want to fail the request, and not to have such linkage if IAL2 check fails) So I didn't mentioned the IAL2 everywhere because that is the default assumption. 
Updated the comments to explain this little bit.